client/components/dataviews/style.scss

```diff
@@ -153,6 +153,11 @@
 			}
 		}
 	}
+
+	// Color overrides for focus states of Action button
+	.components-button:focus:not(:disabled) {
```

Comment:
I moved this style here since it can be shared between Dotcom and A4A. Context: pbxlJb-6A9-p2#comment-4010